lib/ansible/modules/system/filesystem.py

```diff
@@ -16,21 +16,22 @@
 author:
 - Alexander Bulimov (@abulimov)
 module: filesystem
-short_description: Makes file system on block device
+short_description: Makes filesystem
 description:
-  - This module creates file system.
+  - This module creates filesystem.
```

Comment:
`This module creates a file system`

Reply:
Done (using: `This module creates a filesystem.`)